### PR TITLE
webgpu: Update readme and simplify flags

### PR DESCRIPTION
**Pull request by qjia7**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

### CODE REVIEW

File: tfjs-backend-webgpu/README.md [hunk 1]
```diff
@@ -15,8 +15,7 @@ the following models:
 - Speech commands
 
 Note that WebGPU hasn't been officially supported by browsers. Only Google
```
Comment (gyagp) [inline tfjs-backend-webgpu/README.md:17]: This sentence also needs a change.

Comment (gyagp) [reply tfjs-backend-webgpu/README.md:17]: Maybe you can change the whole paragraph to "Google Chrome started to support WebGPU by default in M113 on May 2, 2023".

File: tfjs-backend-webgpu/README.md [hunk 2]
```diff
@@ -81,9 +80,10 @@ yarn build
 ```
 
 ## Testing
-Currently the Canary channel of Chrome is used for testing of the WebGPU
+Currently the Google Chrome is used for testing of the WebGPU. Please make sure
+the chrome version >= 113.
```
Comment (gyagp) [inline tfjs-backend-webgpu/README.md:84]: This sentence can be removed.

File: tfjs-backend-webgpu/README.md [hunk 3]
```diff
@@ -81,9 +80,10 @@ yarn build
...
 backend:
 
 ```sh
-yarn test  # --test_env=CHROME_CANARY_BIN=/path/to/chrome
+yarn test # --test_env=CHROME_BIN=/path/to/chrome
```
Comment (gyagp) [inline tfjs-backend-webgpu/README.md:88]: We should continue to stick with Chrome Canary for tests.

**Review by gyagp — APPROVED** [review]
LGTM

**Review by mattsoulanille — APPROVED** [review]
LGTM!

**Pull request merged**